docs/common/logger.rst

```diff
@@ -28,5 +28,53 @@ Available formats are ``["stdout", "csv", "log", "tensorboard", "json"]``.
   model.set_logger(new_logger)
   model.learn(10000)
 
+
+Explanation of logger output
+----------------------------
+
+You can find below short explanations of the value logged in Stable-Baselines3 (SB3).
+Depending on the algorithm used and of the wrappers/callbacks applied, SB3 only logs a subset of those keys during training.
+
+eval/
+^^^^^
+- ``mean_ep_length``: Mean episode length
+- ``mean_reward``: Mean episodic reward (during evaluation), an ``EvalCallback`` is required to compute that value
+- ``success_rate``: Mean success rate during evaluation (1.0 means 100% success), the environment info dict must contain an ``is_success`` key to compute that value
+
+rollout/
+^^^^^^^^
+- ``ep_len_mean``: Mean episode length (averaged over 100 episodes)
+- ``ep_rew_mean``: Mean episodic training reward (averaged over 100 episodes), a ``Monitor`` wrapper is required to compute that value
```

Comment:
From a statistical perspective, `median` is a more robust measure than `mean` for averaging some numeric values. I think episodic length mean's and episodic reward means could be measured in terms of median. This way, the average value will not be susceptible to high reward/length values.

Reply:
That's a bit offtopic w.r.t. that PR but to answer you, the training reward is only a proxy of the performance and should not be used for evaluation/ proper comparison (that's what we have the `EvalCallback` that also reports standard deviation). And median is also not a perfect metric as it may not reflect enough the distribution of scores (you can read more about evaluation [here](https://araffin.github.io/post/rliable/) where IQM is suggested as a replacement), for instance you can change half the scores to zero without changing the median.